examples/antenna_wire_dipole_fs.py

```diff
@@ -14,45 +14,42 @@
 
 fn = Path(__file__)
 
-title = gprMax.Title(name=fn.with_suffix('').name)
-#title: antenna_wire_dipole_fs
+title = gprMax.Title(name=fn.with_suffix("").name)
+# title: antenna_wire_dipole_fs
 
 domain = gprMax.Domain(p1=(0.050, 0.050, 0.200))
-#domain: 0.050 0.050 0.200
+# domain: 0.050 0.050 0.200
 
 dxdydz = gprMax.Discretisation(p1=(0.001, 0.001, 0.001))
-#dx_dy_dz: 0.001 0.001 0.001
+# dx_dy_dz: 0.001 0.001 0.001
 
 time_window = gprMax.TimeWindow(time=10e-9)
-#time_window: 10e-9
+# time_window: 10e-9
 
-waveform = gprMax.Waveform(wave_type='gaussian', amp=1, freq=1e9, id='mypulse')
-#waveform: gaussian 1 1e9 mypulse
+waveform = gprMax.Waveform(wave_type="gaussian", amp=1, freq=1e9, id="mypulse")
+# waveform: gaussian 1 1e9 mypulse
 
-transmission_line = gprMax.TransmissionLine(polarisation='z',
-                                            p1=(0.025, 0.025, 0.100),
-                                            resistance=73,
-                                            waveform_id='mypulse')
-#transmission_line: z 0.025 0.025 0.100 73 mypulse
+transmission_line = gprMax.TransmissionLine(
+    polarisation="z", p1=(0.025, 0.025, 0.100), resistance=73, waveform_id="mypulse"
+)
+# transmission_line: z 0.025 0.025 0.100 73 mypulse
 
 ## 150mm length wire
-e1 = gprMax.Edge(p1=(0.025, 0.025, 0.025),
-                 p2=(0.025, 0.025, 0.175),
-                 material_id='pec')
-#edge: 0.025 0.025 0.025 0.025 0.025 0.175 pec
+e1 = gprMax.Edge(p1=(0.025, 0.025, 0.025), p2=(0.025, 0.025, 0.175), material_id="pec")
+# edge: 0.025 0.025 0.025 0.025 0.025 0.175 pec
 
 ## 1mm gap at centre of dipole
-e2 = gprMax.Edge(p1=(0.025, 0.025, 0.100),
-                 p2=(0.025, 0.025, 0.101),
-                 material_id='free_space')
-#edge: 0.025 0.025 0.100 0.025 0.025 0.101 free_space
-
-gv = gprMax.GeometryView(p1=(0.020, 0.020, 0.020),
-                         p2=(0.030, 0.030, 0.180),
-                         dl=(0.001, 0.001, 0.001),
-                         filename=fn.with_suffix('').name,
-                         output_type='n')
-#geometry_view: 0.020 0.020 0.020 0.030 0.030 0.180 0.001 0.001 0.001 antenna_wire_dipole_fs f
+e2 = gprMax.Edge(p1=(0.025, 0.025, 0.100), p2=(0.025, 0.025, 0.101), material_id="free_space")
+# edge: 0.025 0.025 0.100 0.025 0.025 0.101 free_space
+
+gv = gprMax.GeometryView(
+    p1=(0.020, 0.020, 0.020),
+    p2=(0.030, 0.030, 0.180),
+    dl=(0.001, 0.001, 0.001),
+    filename=fn.with_suffix("").name,
+    output_type="n",
+)
+# geometry_view: 0.020 0.020 0.020 0.030 0.030 0.180 0.001 0.001 0.001 antenna_wire_dipole_fs f

```

Comment:
Note that a whitespace is being inserted here which we don't want since this is a hash command and not a Python comment. Occurs in other places also and will need checked.

Reply:
Ok, sir. I will look into it and try to resolve the problem.

Reply:
> Note that a whitespace is being inserted here which we don't want since this is a hash command and not a Python comment. Occurs in other places also and will need checked.

@craig-warren sir, are you saying about the whitespace being inserted after the `#` character in `.py` files only? Because whitespace is not being inserted after the `#` character in any of the `.in` files. If you don't want any whitespace to be introduced after the `#` character in any of the `.py` files also, then I will look into it further.

Reply:
@Sai-Suraj-27 yes it is the hash character in the ``.py`` that seems to be effected, because it denotes a comment in Python and not a gprMax hash command. Its not a big deal, so don't spend too much time trying to fix it.

Reply:
Yes, sir👍. Please review the rest of the code and tell me if there are any changes required, thank you.